.github/workflows/test-suite-native-e2e-android.yml

```
@@ -278,6 +279,14 @@ jobs:
             --framework-version=v11.2.0
           npx currents upload --project-id=${CURRENTS_PROJECT_ID} --key=${CURRENTS_RECORD_KEY} --ci-build-id=${CURRENTS_CI_BUILD_ID} --report-dir ./currents
 
+      - name: Post Currents link
+        if: ${{ ! cancelled() }} && github.event_name == 'pull_request'
```

actionlint (1.7.4)

if: condition "${{ ! cancelled() }} && github.event_name == 'pull_request'" is always evaluated to true because extra characters are around ${{ }}

(if-cond)